src/components/__tests__/Form.test.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * @vitest-environment happy-dom
+ */
+
+import { type VueWrapper, mount } from "@vue/test-utils";
+import { describe, it, expect, beforeEach } from "vitest";
+import Form from "../Form.vue";
+
+describe("Form", () => {
+  let wrapper: VueWrapper;
+
+  beforeEach(() => {
+    wrapper = mount(Form);
+  });
+
+  it("Should render correct form and empty list", () => {
+    expect(wrapper.find("form").exists()).toBeTruthy();
+    expect(wrapper.findAll("input").length).toBe(2);
+    expect(wrapper.find("button").exists()).toBeTruthy();
+
+    const nameList = wrapper.find("ul");
+    expect(nameList.exists()).toBeTruthy();
+    expect(nameList.findAll("li").length).toBe(0);
+  });
+
+  it("Input should change text", async () => {
+    const [firstNameInput, secondNameInput] = wrapper.findAll("input");
+
+    await firstNameInput.setValue("John");
```

Comment:
this test can't fail if there are inputs.
is should be handled by emitting events